src/supertux/autotile_parser.cpp

```diff
@@ -45,7 +45,7 @@ AutotileParser::parse()
   auto root = doc.get_root();
 
   if (root.get_name() != "supertux-autotiles") {
-    throw std::runtime_error("file is not a supertux autotile configuration file.");
+    throw std::runtime_error("File is not a supertux autotile configuration file.");
```

Comment:
Should be indicated such format is required to be "supertux-autotiles" somewhere in the error message.